### PR TITLE
HARP-5699 Adding three.js raycast demo

### PR DESCRIPTION
**Pull request by nzjony**
Signed-off-by: Jonathan Stichbury <2533428+nzjony@users.noreply.github.com>

### CODE REVIEW

File: @here/harp-examples/README.md [hunk 1]
```diff
@@ -43,3 +42,8 @@ We can group the examples by several categories, looking at what kind of feature
 ### Styling
 
 1. [Display three map views, side by side](src/multiview_triple-view.ts), in which we show the a map with three different styles at the same time, using OMV Data.
+
+### [three.js](https://github.com/mrdoob/three.js/)
```
Comment (musculman) [inline @here/harp-examples/README.md]: Please use https://threejs.org/ instead

Comment (nzjony) [reply @here/harp-examples/README.md]: Done.

**Comment by nzjony** [comment]
Please wait to review, I see there is a method in mapview which will do this work for us. So I will rework this demo.

**Comment by nzjony** [comment]
@heremaps-bot retest this please

**Comment by nzjony** [comment]
@heremaps-bot retest this please

**Pull request merged**